### ISSUE TITLE
Nothing happens when launching AppImage version of Localsend

### ISSUE THREAD

**Issue by jerrylususu** [localsend/localsend#30]
As a workaround of issue #29 , I tried to use Chrome OS's Linux container and run Localsend's AppImage version, to see if port mapping can make Localsend correctly discover local devices. Yet it seems when I launch it, nothing happens. `htop` shows the process has been started, but there is no UI, no messages, no stdout printing. It would be great if you can let me know how can I diagnose this problem or enable some verbose options to see what's going on.

![image](https://user-images.githubusercontent.com/17522475/215012508-0ac41c3d-87d0-486f-9fdb-4de68618005f.png)
![image](https://user-images.githubusercontent.com/17522475/215012555-dba8236e-3c7e-4e96-89c7-75e11b529bad.png)

**Comment by Tienisto** [comment]
Actually, when you start the AppImage from command line, then there should be logs

![grafik](https://user-images.githubusercontent.com/38380847/215091564-6e483d50-088c-47f2-8f98-c40efb3e2f43.png)

**Comment by jerrylususu** [comment]
That's interesting. I tried to run this AppImage with `strace` to get some insight on what's happening. It seems a lot of `openat` calls failed with `ENOENT`, so perhaps that's some dependency missing? I am not very experienced in Linux app development, so this is just a guess. You can try to diff my log to a log generated when the AppImage version is running as expected. Hope this helps.

[strace.log](https://github.com/localsend/localsend/files/10524435/strace.log)

**Comment by jerrylususu** [comment]
I just performed a restart and it's working now. I suppose there might be some temporary problem or incorrect state that got resolved by the restart. Though since the Linux container in Chrome OS is just another container with its seperate network stack and subnet, discovering local devices is still not working. But nonetheless it's indeed working now, so I am closing this issue. Thanks again for your great creation!
